kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/util/StableKroxyliciousLinkGenerator.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.internal.util;
+
+import io.kroxylicious.proxy.VersionInfo;
+
+public class StableKroxyliciousLinkGenerator {
+
+    public static String errorLink(String slug) {
+        return generateLink("errors", slug);
+    }
+
+    public static String generateLink(String namespace, String slug) {
+        return String.format("https://kroxylicious.io/redirects/%s/%s/%s", namespace, VersionInfo.VERSION_INFO.version(), slug);
```

Comment:
I hesitated about making the domain configurable as well, for forks etc but that felt like going a bit far. I'm open to persuasion however.